### PR TITLE
[doc][yba] DR Managing tables and indexes fixes

### PR DESCRIPTION
**Pull request by ddhodge**
Clarify alter with constraint steps
DOC-389

@netlify /preview/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-assist.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ menu:
 type: docs
 ---
 
+{{<tip title="v2.20 and earlier">}}
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-assist.md:15]: The "v" in v2.20 gets capitalized, not such a big deal.. looks a bit odd. Maybe change it for "For v2.20 and earlier"

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md [hunk 2]
```diff
@@ -22,11 +22,24 @@ You should perform these actions in a specific order, depending on whether perfo
...
+| 1. CREATE INDEX | 2. CREATE INDEX | 3. [Reconcile](#reconcile-configuration) |
+| 2. DROP INDEX   | 1. DROP INDEX   | 3. [Reconcile](#reconcile-configuration) |
 | 1. CREATE TABLE foo PARTITION OF bar | 2. CREATE TABLE foo PARTITION OF bar | 3. Add the table to replication |
 | 2. ALTER TABLE or INDEX | 1. ALTER TABLE or INDEX | No changes needed |
+| 1. ALTER TABLE ADD CONSTRAINT UNIQUE | 2. ALTER TABLE ADD CONSTRAINT UNIQUE | 3. [Reconcile](#reconcile-configuration) |
+| 2. ALTER TABLE DROP CONSTRAINT (unique constraints only) | 1. ALTER TABLE DROP CONSTRAINT (unique constraints only) | 3. [Reconcile](#reconcile-configuration) |
+
+| DDL | Step 1 | Step 2 |  Step 3 |
+| :--- | :--- | :--- | :--- |
+| CREATE TABLE | Execute on Primary | Execute on Replica | Add table to replication |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md:35]: change standby to replica and add links to step 3 wherever applicable.

**Review by aishwarya24 — COMMENTED** [review]
some comments.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md [hunk 3]
```diff
@@ -19,14 +19,16 @@ When DDL changes are made to databases in replication for xCluster disaster reco
...
-| 1. CREATE INDEX | 2. CREATE INDEX | 3. [Resynchronize](#resynchronize-yba) |
-| 2. DROP INDEX   | 1. DROP INDEX   | 3. [Resynchronize](#resynchronize-yba) |
-| 1. CREATE TABLE foo PARTITION OF bar | 2. CREATE TABLE foo PARTITION OF bar | 3. Add the table to replication |
-| 2. ALTER TABLE or INDEX | 1. ALTER TABLE or INDEX | No changes needed |
+| DDL | Step 1 | Step 2 | Step 3 |
+| :-- | :----- | :----- | :----- |
+| CREATE TABLE | Execute on Primary | Execute on Replica | [Add table to replication](#add-a-table-to-dr) |
+| DROP TABLE   | [Remove table from replication](#remove-a-table-from-dr) | Execute on Replica | Execute on Primary |
+| CREATE TABLE foo<br>PARTITION OF bar | Execute on Primary | Execute on Replica | [Add table to replication](#add-a-table-to-dr) |
+| CREATE INDEX | Execute on Primary | Execute&nbsp;on&nbsp;Replica | [Reconcile](#reconcile-configuration) |
```
Comment (shahrooz1997) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md:27]: nit: change `Resynchronize` -> `Reconcile configuration`

File: docs/content/stable/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md [hunk 4]
```diff
@@ -19,14 +19,16 @@ When DDL changes are made to databases in replication for xCluster disaster reco
...
-| 2. DROP INDEX   | 1. DROP INDEX   | 3. [Resynchronize](#resynchronize-yba) |
-| 1. CREATE TABLE foo PARTITION OF bar | 2. CREATE TABLE foo PARTITION OF bar | 3. Add the table to replication |
-| 2. ALTER TABLE or INDEX | 1. ALTER TABLE or INDEX | No changes needed |
+| DDL | Step 1 | Step 2 | Step 3 |
+| :-- | :----- | :----- | :----- |
+| CREATE TABLE | Execute on Primary | Execute on Replica | [Add table to replication](#add-a-table-to-dr) |
+| DROP TABLE   | [Remove table from replication](#remove-a-table-from-dr) | Execute on Replica | Execute on Primary |
+| CREATE TABLE foo<br>PARTITION OF bar | Execute on Primary | Execute on Replica | [Add table to replication](#add-a-table-to-dr) |
+| CREATE INDEX | Execute on Primary | Execute&nbsp;on&nbsp;Replica | [Reconcile](#reconcile-configuration) |
+| DROP INDEX   | Execute on Replica | Execute on Primary | [Reconcile](#reconcile-configuration) |
```
Comment (shahrooz1997) [inline docs/content/stable/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md:28]: The same as above

File: docs/content/v2.20/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md [hunk 5]
```diff
@@ -19,14 +19,16 @@ When DDL changes are made to databases in replication for xCluster disaster reco
...
-| 2. DROP TABLE   | 3. DROP TABLE   | 1. Remove the table from replication. |
-| 1. CREATE INDEX | 2. CREATE INDEX | 3. [Resynchronize](#resynchronize-yba) |
-| 2. DROP INDEX   | 1. DROP INDEX   | 3. [Resynchronize](#resynchronize-yba) |
-| 1. CREATE TABLE foo PARTITION OF bar | 2. CREATE TABLE foo PARTITION OF bar | 3. Add the table to replication |
-| 2. ALTER TABLE or INDEX | 1. ALTER TABLE or INDEX | No changes needed |
+| DDL | Step 1 | Step 2 | Step 3 |
+| :-- | :----- | :----- | :----- |
+| CREATE TABLE | Execute on Primary | Execute on Replica | [Add table to replication](#add-a-table-to-dr) |
+| DROP TABLE   | [Remove table from replication](#remove-a-table-from-dr) | Execute on Replica | Execute on Primary |
+| CREATE TABLE foo<br>PARTITION OF bar | Execute on Primary | Execute on Replica | [Add table to replication](#add-a-table-to-dr) |
```
Comment (shahrooz1997) [inline docs/content/v2.20/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md:26]: same as above

**Review by shahrooz1997 — APPROVED** [review]
LGTM. Added one small comment.

**Pull request merged**